azure/kubernetes/aks-single-region/main.tf

```diff
@@ -1,5 +1,5 @@
 resource "azurerm_resource_group" "app_rg" {
-  name     = var.resource_group_name
+  name     = "${var.resource_prefix}-rg"
```

Comment:
I'm confused about this one.
Aren't you actively defining this outside already, so why rely here on the prefix again?

Meaning if someone would change the `resource_group_name` value in the test, it would still be `rg` in here and may fail at other steps.

But maybe I'm missing something on how exactly it's done.